### PR TITLE
Update delegation-pool-operations.md

### PR DESCRIPTION
**Pull request by michelle-aptos**
updated for missing --address param

### CODE REVIEW

File: developer-docs-site/docs/nodes/validator-node/operator/delegation-pool-operations.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ To create a delegation pool and obtain information about it, [connect to the Apt
 You can use the following CLI commands to obtain the delegation pool address depending on where you are in the process:
 - Before you create the delegation pool:  
     ```bash
-    aptos account derive-resource-account-address --seed "aptos_framework::delegation_pool<SEED>" --seed-encoding utf8
+    aptos account derive-resource-account-address --address --seed "aptos_framework::delegation_pool<SEED>" --seed-encoding utf8
```
Comment (gregnazario) [inline developer-docs-site/docs/nodes/validator-node/operator/delegation-pool-operations.md:29]: Please add `<OPERATOR_ADDRESS>` or whichever address is supposed to be derived from

Comment (clay-aptos) [reply developer-docs-site/docs/nodes/validator-node/operator/delegation-pool-operations.md]: Done!

**Review by clay-aptos — APPROVED** [review]
Thanks for making the update, Michelle! And thanks for the review, Greg!

**Review by michelle-aptos — COMMENTED** [review]
I think it should be <owner_address>

**Comment by clay-aptos** [comment]
> I think it should be <owner_address>

Thanks! Done.

**Review by saharct — APPROVED** [review]

**Review by gregnazario — APPROVED** [review]

**Pull request merged**